### PR TITLE
[admin] Add tooltips for the room visibility and clickability icons

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Resolves:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1019

Description:
sparkle-buttons-title-02 sparkle-buttons-title-01

### CODE REVIEW

File: src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ import { useUser } from "hooks/useUser";
 
 import { RenderMarkdown } from "components/organisms/RenderMarkdown";
 
-import { ButtonNG } from "components/atoms/ButtonNG/ButtonNG";
```
Comment (denisdimitrov) [inline src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx:34]: Nice!

File: src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.tsx [hunk 2]
```diff
@@ -101,12 +101,14 @@ export const RoomCard: React.FC<RoomCardProps> = ({
             iconName={room.isEnabled ? faEye : faEyeSlash}
             disabled={isTogglingRoom}
             onClick={toggleRoom}
+            title={`click to ${room.isEnabled ? "hide" : "show"} room`}
```
Comment (denisdimitrov) [inline src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.tsx:104]: The prop is `title` but the visual feedback is a tooltip. I suppose you are using the default functionality that displays an elements title as tooltip?

Comment (goran-peoski-work) [reply src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.tsx:104]: Default for now. Can update if new design provided.

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**